.vscode/launch.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
These files need to be re-added. If this branch is merged the it will remove these files.